app/templates/suppliers/registered_address.html

```diff
@@ -0,0 +1,126 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/forms/macros/forms.html" as forms %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block head %}
+  <link type="text/css" rel="stylesheet" media="screen" href="{{ asset_fingerprinter.get_url('stylesheets/location-autocomplete.min.css') }}"/>
+  {{ super() }}
+{% endblock %}
+{% block page_title %}Registered address – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Supplier details"
```

Comment:
I think we're going for "Company details" as the breadcrumb now. That's what's in the prototype kit at the moment. Kev's going to make a ticket to go through and make sure everything else is updated, but we can probably do this bit now.

Reply:
https://trello.com/c/yxK7OIOC/83-update-breadcrumbs-for-supplier-details-to-company-details

Reply:
Cool, have changed.